plugins/star-rating/api/api.js

```diff
@@ -66,7 +66,7 @@ var exported = {},
             common.db.collection(collectionName).insert(widget, function(err) {
                 if (!err) {
                     common.returnMessage(ob.params, 201, "Success");
-                    plugins.dispatch("/systemlogs", {params: params, action: "Widget added", data: widget});
+                    plugins.dispatch("/systemlogs", {params: params, action: "Feedback widget added", data: widget});
```

Comment:
Can we better have them as localized strings, so the code here would not actually change in future, but rather only localized string changes here:
https://github.com/Countly/countly-server/blob/master/plugins/systemlogs/frontend/public/localization/systemlogs.properties#L58